msal/src/main/java/com/microsoft/identity/client/internal/authorities/Authority.java

```diff
@@ -44,7 +44,7 @@
     private static final String TAG = Authority.class.getSimpleName();
 
     private static final String ADFS_PATH_SEGMENT = "adfs";
-    private static final String B2C_PATH_SEGMENT = "b2c";
+    private static final String B2C_PATH_SEGMENT = "tfp";
```

Comment:
This part LGTM

Reply:
mAuthorityTypeString in the AzureActiveDirectoryB2CAuthority (https://github.com/AzureAD/microsoft-authentication-library-for-android/blob/a88cf0e6b7a4f1b6fa52363fb1044c1d4b422301/msal/src/main/java/com/microsoft/identity/client/internal/authorities/AzureActiveDirectoryB2CAuthority.java#L37) does this need to change too?

Reply:
No, it needn't. This should be the same with the authority type in config file. @shoatman Please confirm.

Reply:
Yes this should be tfp.  Looks like b2c was set in error... i'm assuming by me ;-)

Reply:
Hi @shoatman sorry, I'm bit confused. In the config.json, for the B2C authority, should we lable the authority type as "B2C" or "tfp"?

I think "B2C" is a more seasonable type. Because there are two kinds of authority schema for B2C scenario. "tfp" is just one of them. I've asked @danieldobalian for the B2C wiki link.